package.json

```diff
@@ -151,7 +164,54 @@
     "integration-tests"
   ],
   "jest": {
-    "clearMocks": true,
-    "resetMocks": false
+    "roots": [
```

Comment:
This configuration was the result of ejecting out of CRA (for purposes of running jest unit tests only) and deleting things that aren't relevant to us.